document/core/binary/modules.rst

```diff
@@ -315,14 +321,23 @@ The *element section* has the id 9.
 It decodes into a vector of :ref:`element segments <syntax-elem>` that represent the |MELEM| component of a :ref:`module <syntax-module>`.
 
 .. math::
-   \begin{array}{llclll}
+   \begin{array}{llclll@{\qquad}l}
    \production{element section} & \Belemsec &::=&
      \X{seg}^\ast{:}\Bsection_9(\Bvec(\Belem)) &\Rightarrow& \X{seg} \\
    \production{element segment} & \Belem &::=&
-     x{:}\Btableidx~~e{:}\Bexpr~~y^\ast{:}\Bvec(\Bfuncidx)
-       &\Rightarrow& \{ \ETABLE~x, \EOFFSET~e, \EINIT~y^\ast \} \\
+     \hex{00}~~e{:}\Bexpr~~y^\ast{:}\Bvec(\Bfuncidx)
```

Comment:
Hm, how would would this scale to multiple tables then?

Reply:
The plan is to use that byte for flags, one bit of which will indicate that a table index varint follows.